src/layouts/common.scss

```diff
@@ -81,6 +80,12 @@ html, body, #___gatsby {
   min-height: 100%;
   position: relative;
   box-sizing: border-box;
+  background-color: $blue;
+
+  &.bg-grey {
+    background-color: $grey-bg;
+  }
+
   &-body {
     h1, p {
```

Comment:
and here

Reply:
It's global styles for `h1` and `p`, i suppose that some code uses a color definition based on common CSS